apps/frontend/src/app/PageCharacter/CharacterDisplay/Tabs/TabOptimize/index.tsx

```diff
@@ -48,14 +48,20 @@ import WorkerErr from './Components/WorkerErr';
 import { compactArtifacts, dynamicData } from './foreground';
 import useBuildResult from './useBuildResult';
 import useBuildSetting from './useBuildSetting';
+import NotificationsOffIcon from '@mui/icons-material/NotificationsOff';
+import NotificationsActiveIcon from '@mui/icons-material/NotificationsActive';
 
+const mp3_url = 'https://media.geeksforgeeks.org/wp-content/uploads/20190531135120/beep.mp3';
```

Comment:
can we package a different sound this is horrendous lol

Reply:
LOL